helix-core/src/main/java/org/apache/helix/controller/GenericHelixController.java

```diff
@@ -220,17 +221,24 @@ public GenericHelixController(String clusterName, Set<Pipeline.Type> enabledPipe
   class RebalanceTask extends TimerTask {
     final HelixManager _manager;
     final ClusterEventType _clusterEventType;
+    private final boolean _requestCacheRefresh;
```

Comment:
Please follow a proper boolean naming convention. In this case, `shouldRefreshCache` or something like this would read much better.

Reply:
Sounds good. BTW, could you please also paste this convention so I can take a look?